assistant/src/test/java/com/ibm/watson/assistant/v2/AssistantServiceIT.java

```diff
@@ -15,20 +15,8 @@
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertTrue;
 
-import com.ibm.watson.assistant.v2.model.CreateSessionOptions;
-import com.ibm.watson.assistant.v2.model.DeleteSessionOptions;
-import com.ibm.watson.assistant.v2.model.MessageContext;
-import com.ibm.watson.assistant.v2.model.MessageContextStateless;
-import com.ibm.watson.assistant.v2.model.MessageInput;
-import com.ibm.watson.assistant.v2.model.MessageInputOptions;
-import com.ibm.watson.assistant.v2.model.MessageInputOptionsStateless;
-import com.ibm.watson.assistant.v2.model.MessageInputStateless;
-import com.ibm.watson.assistant.v2.model.MessageOptions;
-import com.ibm.watson.assistant.v2.model.MessageResponse;
-import com.ibm.watson.assistant.v2.model.MessageResponseStateless;
-import com.ibm.watson.assistant.v2.model.MessageStatelessOptions;
-import com.ibm.watson.assistant.v2.model.RuntimeResponseGeneric;
-import com.ibm.watson.assistant.v2.model.SessionResponse;
+import com.ibm.watson.assistant.v2.model.*;
```

Comment:
I have a feeling that the linter we use will not like this

Reply:
I ran the google format here [c24a60278b513219da1c3a233f9b83f8f23d0961](https://github.com/watson-developer-cloud/java-sdk/pull/1128/commits/c24a60278b513219da1c3a233f9b83f8f23d0961)